src/resources/postcss/tickets-admin/settings/_tickets-commerce.pcss

```diff
@@ -496,6 +506,36 @@
 	text-align: center;
 	text-decoration: none;
 	width: 233px;
+
+	&:active,
+	&:hover {
+		background-color: var(--tec-color-button-primary-hover);
+		color: #fff;
+	}
+
+	&:active {
+		background-color: var(--tec-color-button-primary-active);
+	}
+}
+
+.tec-tickets__admin-settings-tickets-commerce-gateways-item-button-link--active {
+	background-color: #fff;
+	border-color: #3c434a;
+	border-style: solid;
+	border-width: 1px;
```

Comment:
We have a variable for this border with.

Reply:
If we do, I don't see it. Which file are you looking at?

Reply:
@codingmusician You are right, we don't! I would have bet that we had one here: https://github.com/the-events-calendar/tribe-common-styles/blob/main/variables/_borders.pcss